source/_components/light.decora_wifi.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "Decora Wi-Fi"
+description: "Instructions on how to setup Leviton Decora Smart Wi-Fi switches/dimmers within Home Assistant."
+date: 2017-07-19 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_iot_class: "Local Polling"
+logo: leviton.png
+ha_release: 0.37
```

Comment:
ha_release will need to be updated to a newer version. 0.51 is the most likely next release that this could be integrated into.